app/assets/stylesheets/contacts.scss

```diff
@@ -12,6 +12,11 @@
     }
   }
 
+  .sidebar{
```

Comment:
- Merge rule `.sidebar` with rule on line 2
- Opening curly brace `{` should be preceded by one space